third-party/CMakeLists.txt

```diff
@@ -161,8 +161,8 @@ set(UNIBILIUM_SHA256 29815283c654277ef77a3adcc8840db79ddbb20a0f0b0c8f648bd8cd49a
 set(LIBTERMKEY_URL http://www.leonerd.org.uk/code/libtermkey/libtermkey-0.22.tar.gz)
 set(LIBTERMKEY_SHA256 6945bd3c4aaa83da83d80a045c5563da4edd7d0374c62c0d35aec09eb3014600)
 
-set(LIBVTERM_URL https://github.com/neovim/libvterm/archive/65dbda3ed214f036ee799d18b2e693a833a0e591.tar.gz)
-set(LIBVTERM_SHA256 95d3c7e86336fbd40dfd7a0aa0a795320bb71bc957ea995ea0e549c96d20db3a)
+set(LIBVTERM_URL http://www.leonerd.org.uk/code/libvterm/libvterm-0.1.4.tar.gz)
+set(LIBVTERM_SHA256 bc70349e95559c667672fc8c55b9527d9db9ada0fb80a3beda533418d782d3dd)
```

Comment:
yikes, no https? I really wish git-bzr did not mess up the hashes, that is the main problem with https://github.com/neovim/libvterm as a mirror .

Reply:
No, `leonerd.org.uk` doesn't support https... (I checked for #12797)

What is the issue with the hashes? That they have to be recomputed manually?

Reply:
> What is the issue with the hashes? That they have to be recomputed manually?

somehow git-bzr changes the hashes every time we update the mirror: https://github.com/neovim/bot-ci/blob/c799c16b003db67e353542ee7f1494731931dc6b/ci/sync-mirrors.sh#L46

not sure if we're using git-bzr wrong or if bzr just sucks.

Reply:
actually it seems like that maybe isn't happening anymore ? if so then there's no reason to use leonerd.org.uk ...

Reply:
except that the mirror isn't updated yet...

Reply:
Ups, I did not notice the missing `s` in `http`. I'll update the PR to use https://github.com/neovim/libvterm/commit/54c03b21f763fa775a4c0643a9d8326342873179.

Reply:
@qvacua the bump was already merged in #12797, by the way, so you can just rebase.